src/modules/main/components/EncourageSection/EncourageSection.jsx

```diff
@@ -1,21 +1,48 @@
 import styles from "./encourage_section.module.scss";
-import dog from "../../../../assets/image/encourages/dog.png";
+import dog2 from "../../../../assets/image/encourages/dog2.png";
```

Comment:
Wouldn't it be easier to call it `encourageSectionImage`? I think it will be more verbose in this case